yaksh/views.py

```diff
@@ -580,7 +595,7 @@ def add_course(request):
 def enroll_request(request, course_id):
     user = request.user
     ci = RequestContext(request)
-    course = get_object_or_404(Course, pk=course_id)
+    course = get_object_or_404(Course, pk=course_id, is_trial=False)
```

Comment:
There are two ways to go about this -
1. We can override django's queryset api and force it to always return only a ```is_trial=False``` queryset object. But the problem is its not a good thing to tinker around django's api. 
2. We can create a manager method but it seems quite unnecessary since most of the ```is_trial=False``` filter calls have been made only once in views.

So I have kept some ```is_trial``` while a few unecessary ones I have deleted.